plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/PydevConsoleConstants.java

```diff
@@ -85,6 +89,29 @@
             { "OpenGL (glut)", "glut" },
             { "pyglet (pyglet)", "pyglet" } };
 
+    public static final String ACTIVE_EDITOR_INTERPRETER_REPRESENTATION = "active editor";
+    public static final String PYTHON_INTERPRETER_REPRESENTATION = "python";
+    public static final String PYDEV_INTERPRETER_REPRESENTATION = "pydev";
+    public static final String JYTHON_INTERPRETER_REPRESENTATION = "jython";
+    public static final String IRONPYTHON_INTERPRETER_REPRESENTATION = "ironpython";
+    public static final String INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = "INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER";
+    public static final String DEFAULT_INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = "none";
```

Comment:
I think that the default could be an empty string (since you have to treat that anyways). References to the current `"none"` should use the `DEFAULT_INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER` constant.